### PR TITLE
Updated FindWorkspace to get mappings

### PR DESCRIPTION
**Pull request by jpricketMSFT**
It also parses out the team project name from the first mapping

### CODE REVIEW

File: test/tfvc/commands/findworkspace.test.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+"use strict";
+
+import { FindWorkspace } from "../../../src/tfvc/commands/findworkspace";
+import { TfvcError } from "../../../src/tfvc/tfvcerror";
+import { IExecutionResult, IWorkspace } from "../../../src/tfvc/interfaces";
+
+var chai = require("chai");
```
Comment (jeffyoung) [inline test/tfvc/commands/findworkspace.test.ts]: Either before or after going in, we should make the same change as in my last PR (import chai).

File: test/tfvc/commands/findworkspace.test.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+/* tslint:disable:no-unused-variable */
+var expect = chai.expect;
+/* tslint:enable:no-unused-variable */
+var assert = chai.assert;
+chai.should();
+
+describe("FindWorkspaceCommand", function() {
```
Comment (jeffyoung) [inline test/tfvc/commands/findworkspace.test.ts]: Perhaps we should have a prefix like "Tfvc-" on the describe so we can immediately see that it's a part of that extension?

File: src/tfvc/tfvcerror.ts [hunk 3]
```diff
@@ -16,7 +16,18 @@ export class TfvcError {
     tfvcErrorCode: string;
     tfvcCommand: string;
 
-    public constructor(data: ITfvcErrorData) {
+    public constructor(data: ITfvcErrorData | string) {
+        if (typeof data === "string") {
```
Comment (jeffyoung) [inline src/tfvc/tfvcerror.ts]: I'm confused... why allow typeof "string" in the constructor if it's an error?

Comment (jpricketMSFT) [inline src/tfvc/tfvcerror.ts]: I wanted a quick way of creating a "missing argument exception". I agree it is confusing. I will create a static method instead.

File: test/tfvc/commands/findworkspace.test.ts (hunk 1 above)
Comment (jpricketMSFT) [inline test/tfvc/commands/findworkspace.test.ts]: will do

File: test/tfvc/commands/findworkspace.test.ts (hunk 2 above)
Comment (jpricketMSFT) [inline test/tfvc/commands/findworkspace.test.ts]: done

**Pull request merged**